manta-js/package/src/wallet/crate/src/lib.rs

```diff
@@ -1608,6 +1719,35 @@ impl Wallet {
         )
     }
 
+    /// Combine SBT `sign` and `transaction_data` in one call.
+    #[inline]
+    pub fn sign_with_sbt_transaction_data(
```

Comment:
How is this function any different from simply calling `sign_with_transaction_data`?

Reply:
Only the return value is different, I think @ferrell-code  can explain this

Reply:
I think we can remove this

The only difference is the data format and that it converts `TransferPost` into the runtime format. These conversions seem to have been implemented in typescript so I think removing duplicate function makes sense

Reply:
please review this commit: https://github.com/Manta-Network/sdk/pull/114/commits/749abccfa9f515cee54613281c75f90779076781

Reply:
That also works. I just found it absurd to have two different functions doing the same thing.

Reply:
Let me know what you think of fix